src/main/app/claims/models/response/core/courtDetermination.ts

```diff
@@ -0,0 +1,8 @@
+import { PaymentIntention } from 'claims/models/response/core/paymentIntention'
+
+export class CourtDetermination {
```

Comment:
Change to interface if you treat it only as an interface.

Reply:
ok sure